eternalcombat-api/src/main/java/com/eternalcode/combat/region/WorldGuardRegionProvider.java

```diff
@@ -30,33 +30,26 @@ public Location getRegionCenter(Location location) {
         RegionQuery regionQuery = regionContainer.createQuery();
         ApplicableRegionSet applicableRegions = regionQuery.getApplicableRegions(BukkitAdapter.adapt(location));
 
-        double minX = 0;
-        double maxX = 0;
-        double minZ = 0;
-        double maxZ = 0;
+        ProtectedRegion combatRegion = null;
```

Comment:
Maybe use AtomicReference?
```suggestion
        AtomicReference<ProtectedRegion> combatRegion = new AtomicReference<ProtectedRegion>();
```